outputs.tf

```diff
@@ -14,94 +14,94 @@
  * limitations under the License.
  */
 
-output "name" {
-  description = "Cluster name"
-  value       = "${local.cluster_name}"
-}
+// File autogenerated by `helpers/generate_modules/generate_module.py`
 
-output "type" {
-  description = "Cluster type (regional / zonal)"
-  value       = "${local.cluster_type}"
+output "ca_certificate" {
```

Comment:
Why is there a big diff in this file? Shouldn't be changed at all. I'm concerned by the number of unrelated files which are being changed.